python_modules/libraries/dagster-pandas/dagster_pandas_tests/test_pandas_user_error.py

```diff
@@ -3,44 +3,38 @@
 import dagster_pandas as dagster_pd
 import pandas as pd
 import pytest
-from dagster import (
-    DagsterTypeCheckDidNotPass,
-    InputDefinition,
-    OutputDefinition,
-    execute_pipeline,
-    lambda_solid,
-    pipeline,
-)
-from dagster.core.utility_solids import define_stub_solid
+from dagster import DagsterTypeCheckDidNotPass, In, Out, graph, op
 
 
 def test_wrong_output_value():
-    csv_input = InputDefinition("num_csv", dagster_pd.DataFrame)
-
-    @lambda_solid(input_defs=[csv_input], output_def=OutputDefinition(dagster_pd.DataFrame))
-    def test_wrong_output(num_csv):
+    @op(ins={"num_csv": In(dagster_pd.DataFrame)}, out=Out(dagster_pd.DataFrame))
+    def wrong_output(num_csv):
         return "not a dataframe"
 
-    pass_solid = define_stub_solid("pass_solid", pd.DataFrame())
+    @op
+    def pass_df():
+        return pd.DataFrame()
 
-    @pipeline
-    def test_pipeline():
-        return test_wrong_output(pass_solid())
+    @graph
+    def test_graph():
```

Comment:
@dpeng817 after adding execute_in_process for graphs, should we still avoid including "graph" in the name? in case if they are converted to jobs?

Reply:
I think in general we want to avoid calling graphs "graph" in all cases, yea. In general, the pattern is someone would use execute_in_process on the graph for testing, and then still create jobs out of said graph, so in general we want to avoid the name "graph" in graph